yogstation/code/modules/admin/verbs/fix_air.dm

```diff
@@ -30,17 +30,18 @@
 	if(!check_rights(R_ADMIN))
 		return
 
+	if(Master.current_runlevel < RUNLEVEL_GAME)
+		//jamie alert
+		to_chat(src, "This command may not be used before the game has started!", confidential=TRUE)
+		message_admins("[src] has attempted to fix air before the game has started. Please talk some sense into them.")
+		return
+
 	if(alert("Do you want to reset air on the entire z level?", "Fix Air- Z level", "No", "Yes") != "Yes")
 		return
 
 	message_admins("[key_name_admin(usr)] fixed air on zlevel [mob.z]")
 	log_game("[key_name_admin(usr)] fixed air on zlevel [mob.z]")
 
```

Comment:
var/atmos_enabled = SSair.can_fire
	if(atmos_enabled)
		message_admins("Disabling atmospherics to fix air on zlevel")
		SSair.can_fire = FALSE